venues/aclweb.org/ACL/migrate_submissions_ARR_ACL.py

```diff
@@ -0,0 +1,780 @@
+                    "Phonology, Morphology and Word Segmentation",
+                    "Resources and Evaluation",
+                    "Semantics: Lexical",
+                    "Semantics: Sentence-level Semantics, Textual Inference and Other areas",
+                    "Syntax: Tagging, Chunking and Parsing",
+                    "Information Extraction",
+                    "Computational Social Science and Cultural Analytics",
+                    "Information Retrieval and Text Mining",
+                    "Interpretability and Analysis of Models for NLP",
+                    "Machine Translation and Multilinguality",
+                    "NLP Applications",
+                    "Question Answering",
+                    "Dialogue and Interactive Systems",
+                    "Discourse and Pragmatics",
+                    "Generation",
+                    "Language Grounding to Vision, Robotics and Beyond",
+                    "Sentiment Analysis, Stylistic Analysis, and Argument Mining",
+                    "Speech and Multimodality",
+                    "Summarization",
+                    "Special Theme"
+                    ],
+                "required": True,
+                "order": 12
+                },
+            "comment": {
+                "description": "Comment to Senior Area Chairs (500 words)",
+                "order": 13,
+                "value-regex": "[\\S\\s]{0,500}",
+                "required": False
+                }     
+        }
+    }
+)
+
+client.post_invitation(submission_invitation)
+
+# Posting Blind Submission Invitation 
+blind = openreview.Invitation(
+    id = 'aclweb.org/ACL/2022/Conference/-/Blind_Submission',
+    readers = [
+        'everyone'
+        ],
+    writers = [
+        'aclweb.org/ACL/2022/Conference'
+        ],
+    invitees = [
+        'aclweb.org/ACL/2022/Conference'
+        ],
+    signatures = [
+        'aclweb.org/ACL/2022/Conference'
+        ],
+    reply ={
+        "readers" : {
+            "values-regex":".*"
+            },
+        "nonreaders" : {
+            "values-regex":".*"
+            },
+        "writers" : {
+            "values-regex":".*"
+            },
+        "signatures" : {
+            "values":[
+                'aclweb.org/ACL/2022/Conference'
+                ]
+            },
+        "content" : {
+            "authorids" : { 
+                "values-regex": ".*" 
+                },
+            "authors": {
+                "values-regex": ".*" 
+                }
+            }   
+        }
+    )
+
+client.post_invitation(blind)
+
+commitment = openreview.Invitation(
+    id = 'aclweb.org/ACL/2022/Conference/-/Commitment_Submission',
+    signatures = [
+        "aclweb.org/ACL/2022/Conference"
+        ],
+    readers = [
+        "everyone"
+        ],
+    invitees=[
+        "everyone"
+        ],
+    reply={
+        "readers":["aclweb.org/ACL/2022/Conference/Program_Chairs","aclweb.org/ACL/2022/Conference"],
```

Comment:
you need 'values-copied' here inside readers and writers, since {`signatures}` will also be a reader/writer of commitment notes, and signatures would be 
`"signatures": {
        "values-regex": "~.*"
}`
here is the invitation on the live site already: https://openreview.net/invitation/edit?id=aclweb.org/ACL/2022/Conference/-/Commitment_Submission (plus they added a couple more fields)